### PR TITLE
Add Local Government options for LG Users

### PR DESCRIPTION
**Pull request by dhaselhan**
* Replace owners with their own contact name
* Create new Government type Owner
* Update set primary contact to accept multiple types

### CODE REVIEW

File: portal-frontend/src/app/services/authentication/authentication.service.ts [hunk 1]
```diff
@@ -137,4 +140,9 @@ export class AuthenticationService {
   private async getLogoutUrl() {
     return firstValueFrom(this.http.get<{ url: string }>(`${environment.authUrl}/logout/portal`));
   }
+
+  private async loadUser() {
```
Comment (mhuseinov) [inline portal-frontend/src/app/services/authentication/authentication.service.ts:144]: do you need to clean user?

File: services/apps/alcs/src/portal/application-submission/application-owner/application-owner.service.ts [hunk 2]
```diff
@@ -214,16 +214,26 @@ export class ApplicationOwnerService {
     });
   }
 
-  async deleteAgents(application: ApplicationSubmission) {
+  async deleteNonParcelOwners(application: ApplicationSubmission) {
```
Comment (mhuseinov) [inline services/apps/alcs/src/portal/application-submission/application-owner/application-owner.service.ts:217]: maybe I missed it but I do not see unit tests for this

**Review by mhuseinov — COMMENTED** [review]
🚀

File: services/apps/alcs/src/portal/application-submission/application-owner/application-owner.service.ts (hunk 2 above)
Comment (dhaselhan) [reply services/apps/alcs/src/portal/application-submission/application-owner/application-owner.service.ts]: Test added!

**Review by mhuseinov — APPROVED** [review]

**Pull request merged**